packages/testkit/__TESTS__/dropdown.spec.js

```diff
@@ -13,7 +13,9 @@ test.describe("dropdown Class with Storybook", () => {
   });
 
   test("set dropdown value", async ({ page }) => {
-    await page.waitForTimeout(3000);
+    // extended wait for interaction test to finish
+    await page.waitForTimeout(10000);
+    await DropDown.inputField.setText("");
```

Comment:
please notice I've disabled this test for now in master branch, since it caused releases to fail